st2common/st2common/models/api/rule.py

```diff
@@ -162,7 +163,7 @@ def to_model(cls, rule):
 
         trigger = reference.get_str_resource_ref_from_model(trigger_db)
         criteria = dict(getattr(rule, 'criteria', {}))
-        pack = str(rule.pack)
+        pack = getattr(rule, 'pack', DEFAULT_PACK_NAME)
```

Comment:
Yeah, we have the same logic in actions controller right now (defaulting to "default" pack if `pack` is not provided), but I would prefer us to eventually move to `pack` being required.